src/DssSpell.sol

```diff
@@ -38,94 +119,252 @@ contract DssSpellAction is DssAction {
-        // Forum: https://forum.makerdao.com/t/mip101-the-maker-constitution/19621
+
+        // ---------- RWA014-A Onboarding ----------
+        // Poll: https://vote.makerdao.com/polling/QmdRELY7#poll-detail
+        // Forum: https://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384
+
+        onboardRWA014();
+        // Lock RWA014 Token in the URN
+        GemAbstract(RWA014).approve(RWA014_A_URN, 1 * WAD);
+        RwaUrnLike(RWA014_A_URN).lock(1 * WAD);
+
+        // ----- Additional ESM authorization -----
+        DssExecLib.authorize(MCD_JOIN_RWA014_A, ESM);
+        DssExecLib.authorize(RWA014_A_URN, ESM);
+        DssExecLib.authorize(RWA014_A_OUTPUT_CONDUIT, ESM);
+        DssExecLib.authorize(RWA014_A_INPUT_CONDUIT_URN, ESM);
+        DssExecLib.authorize(RWA014_A_INPUT_CONDUIT_JAR, ESM);
+
+        // --------- Keeper Network Amendments ---------
+        // Poll: https://vote.makerdao.com/polling/QmZZJcCj#poll-detail
         // NOTE: ignore in goerli
 
-        // Mip: https://mips.makerdao.com/mips/details/MIP107#6-1-governance-security-engineering-budget
-        // Governance Security Engineering Budget | 2023-05-01 00:00:00 to 2024-04-30 23:59:59 | 2,200,000 DAI | 0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0
+        // GELATO    | 1,500 DAI/day | 3 years | Vest Target: 0x0B5a34D084b6A5ae4361de033d1e6255623b41eD | Treasury: 0xbfDC6b9944B7EFdb1e2Bc9D55ae9424a2a55b206
+        // KEEP3R    | 1,500 DAI/day | 3 years | Vest Target: 0xaeFed819b6657B3960A8515863abe0529Dfc444A | Treasury: 0x4DfC6DA2089b0dfCF04788b341197146Ea97f743
+        // CHAINLINK | 1,500 DAI/day | 3 years | Vest Target: 0xfB5e1D841BDA584Af789bDFABe3c6419140EC065
+        // TECHOPS   | 1,000 DAI/day | 1 years | Vest Target: 0x5A6007d17302238D63aB21407FF600a67765f982
 
-        // Mip: https://mips.makerdao.com/mips/details/MIP107#7-1-multichain-engineering-budget
-        // Multichain Engineering Budget          | 2023-05-01 00:00:00 to 2024-04-30 23:59:59 | 2,300,000 DAI | 0x868B44e8191A2574334deB8E7efA38910df941FA
 
-        // --------- Data Insights MKR Transfer ---------
-        // Mip: https://mips.makerdao.com/mips/details/MIP40c3SP64#mkr-vesting
+        // --------- CAIS Bootstrap Funding ---------
+        // Poll: https://vote.makerdao.com/polling/Qmc6Wqrc#poll-detail
         // NOTE: ignore in goerli
-        // DIN-001 - 103.16 MKR - 0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F
+
+
+        // --------- Onboard GNO to Spark ---------
+        // Poll: https://vote.makerdao.com/polling/QmXdGdxS#poll-detail
+        // Forum: https://forum.makerdao.com/t/onboarding-of-gno-to-spark/20831
+        // List of addresses: https://github.com/marsfoundation/sparklend/blob/master/script/output/5/spark-latest.json
+        {
+            // Whitelist the GNO Fig adapter
+            MedianAbstract(GNO_MEDIANIZER).kiss(SPARK_GNO_ORACLE);
+
+            // Set DAI as a borrowable asset in isolation mode
```

Comment:
Need to tell GovAlpha that the sheet needs to be clearer about this.

Cell says `Borrow Enabled: No` but in isolation mode it's apparently yes (per this line).

Either we need to change the code or the sheet (my guess is that the exec sheet + maybe the doc is badly worded or wrong).

Reply:
Borrow enabled no is for GNO which is the default to be disabled. This line enables DAI as a borrowable isolation mode asset which is another line in the spell sheet.